L/libjulia/common.jl

```diff
@@ -335,7 +335,7 @@ function build_julia(ARGS, version::VersionNumber)
         error("Unsupported Julia version")
     end
 
-    julia_compat = version ≥ v"1.7" ? "1.6" : "1.0"
+    julia_compat = version ≥ v"1.6" ? "1.6" : "1.0"
```

Comment:
Why is this needed in this PR? Seems to belong to the next one?

Reply:
Yes, it is not needed here and will also be in the next PR. Doesn't change anything for this build.